README.md

```diff
@@ -1,5 +1,84 @@
-## OCi8 for ddev
+# DDEV OCI8 Add-on
 
-```ssh
+This add-on installs Oracle Instant Client and the PHP OCI8 extension for DDEV projects, supporting both AMD64 and ARM64 architectures.
+
+## Requirements
+
+- **DDEV** v1.23+
```

Comment:
I would say just say 1.24. And if it's important, add a `ddev_version_constraint` to both the install.yaml and the config.oci8.yaml

```suggestion
- **DDEV** v1.24+
```